_includes/banner.html

```diff
@@ -10,6 +10,10 @@
         <span>&nbsp;&nbsp;</span>
         <a href="/documentation/"><button type="button" class="btn btn-lg btn-primary"><i class="fa fa-book"></i> Documentation</button></a>
         <span>&nbsp;&nbsp;</span>
+        <a href="/resources/"><button type="button" class="btn btn-lg btn-primary"><i class="fa fa-external-link-square"></i> Resources</button></a>
+        <span>&nbsp;&nbsp;</span>
+        <a href="/resources/"><button type="button" class="btn btn-lg btn-primary"><i class="fa fa-rss-square"></i> Ehcache Blog</button></a>
```

Comment:
I agree with the sizing issue. I would indeed put the two blog sections under a single menu item.
For the rest, I believe having doc as a direct link remains interesting.
Could we group resources and downloads under a single menu item?